### ISSUE TITLE
[Suggestion] Right click crafting

### ISSUE THREAD

**Issue by goreae** [Inventory-Tweaks/inventory-tweaks#441]
So in the older versions of inventory tweaks, like MC 1.7.10 and earlier, there was this feature that I loved where you could right click on the output in the crafting window, and you'd get a stack of that output in your hand instead of the one output you'd get from left clicking. This feature was so incredibly useful and I'd love to have it in more recent versions of minecraft.

**Comment by Kobata** [comment]
I'm not 100% sure if it's vanilla or not, but I've frequently seen shift+click craft as much as possible given the number of items in the input.

**Comment by goreae** [comment]
Yes that is a vanilla feature, but for things like pipes, conduits, and torches, I may not want to craft as many as possible. Then I have an inventory full of items I didn't actually want to craft. In that case, I usually just want the stack of items. So I'd do a right click to get just the stack.
